cookbook/core/containerization/Dockerfile_preprocessing

```diff
@@ -0,0 +1,21 @@
+FROM python:3.8-slim-buster
```

Comment:
Can you rename the Dockerfile? I think the best practice would be to name it Dockerfile.<...>. In this case, it'd be `Dockerfile.preprocess` and `Dockerfile.predict`.